examples/managers/mid_agreement_payments.py

```diff
@@ -0,0 +1,113 @@
+import asyncio
+import logging.config
+from datetime import datetime, timedelta
+
+from golem.managers import (
+    Buffer,
+    DefaultAgreementManager,
+    DefaultProposalManager,
+    MidAgreementPaymentsNegotiator,
+    NegotiatingPlugin,
+    PayAllPaymentManager,
+    PaymentPlatformNegotiator,
+    RefreshingDemandManager,
+    SequentialWorkManager,
+    SingleUseActivityManager,
+    WorkContext,
+    WorkResult,
+)
+from golem.node import GolemNode
+from golem.payload import PaymentInfo, RepositoryVmPayload
+from golem.utils.logging import DEFAULT_LOGGING
+
+WORK_RUN_TIME = timedelta(minutes=60)
+
+
+async def print_yagna_payment_status(yagna_path="yagna", network="goerli"):
```

Comment:
I just wonder whether we shouldn't pull this out to some `utils` module within the examples directory...

Reply:
If you import from examples then it is harder to run them without cloning entire repo so I am not sure.